recipes/pudb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2016.2" %}
+
+package:
+  name: pudb
+  version: {{ version }}
+
+source:
+  fn: pudb-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pudb/pudb-{{ version }}.tar.gz
+  md5: 4573b70163329c1cb59836a357bfdf7c
+
+build:
+  # conda skeleton added this, but I'm unsure why it would be needed.
+  # preserve_egg_dir: True
```

Comment:
From my memory, it's conservative, based on any setuptools entry points it sees. My best guess, [this empty GUI scripts](https://github.com/inducer/pudb/blob/f46002a76fac99be1432b9a541aaf464f3e4e397/setup.py#L61) line. If that's really it, then we don't need to worry.